hedera-node/hedera-app-spi/src/testFixtures/java/com/hedera/node/app/spi/fixtures/workflows/ExceptionConditions.java

```diff
@@ -57,6 +74,16 @@ private static Predicate<Throwable> getResponseCodeCheck(@NonNull final Response
         };
     }
 
+    @NonNull
+    private static Predicate<Throwable> getHandleResponseCodeCheck(@NonNull final ResponseCodeEnum responseCode) {
```

Comment:
What do you think about adding the if-statement to `getResponseCodeCheck()` instead of creating new methods?

Reply:
I'm fine with it either way. The only reason I did this was that I tend to make things more granular by default.

Reply:
Agreed with Michael H !

Reply:
Done